es/README.md

```diff
@@ -1,36 +1,70 @@
-## Install, configure and start Elasticsearch
+# Install, configure and start Elasticsearch
 
-### Manual installation
+## Manual installation
 
-Download Elasticsearch (at least 7.6.2 for Geonetwork 3.8.x) from https://www.elastic.co/fr/downloads/elasticsearch
-and copy it to the ES module. eg. es/elasticsearch-7.6.2
+1. Download Elasticsearch (at least 7.6.2 for Geonetwork 4.0.x) from https://www.elastic.co/downloads/elasticsearch
+and copy to the ES module. eg. es/elasticsearch-7.6.2
  
-Start ES using:
+2. Start ES using:
 
-```shell script
-./bin/elasticsearch
-```
+   ```shell script
+   ./bin/elasticsearch
+   ```
 
+3. Check that elasticsearch is running by visiting http://localhost:9200 in a browser
 
-### Maven installation
+## Maven installation
 
-Maven can take care of the installation steps:
-* download
-* initialize collection
-* start
+1. Maven can take care of the installation steps:
 
-Use the following commands:
+   * download
+   * initialize collection
+   * start
 
-```shell script
-cd es
-mvn install -Pes-download
-mvn exec:exec -Des-start
-```
+2. Use the following commands:
+
+   ```shell script
+   cd es
+   mvn install -Pes-download
+   mvn exec:exec -Des-start
+   ```
+3. Check that elasticsearch is running by visiting http://localhost:9200 in a browser
+
+## Docker installation
```

Comment:
Can't we refer to https://github.com/geonetwork/docker-geonetwork/tree/master/4.0.5 or https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html to avoid to add extra docker config file here ?

Reply:
Perhaps, I was only writing up the instructions my co-workers provided (amusingly each one had a different way of running).